de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/SarosServer.java

```diff
@@ -62,6 +63,8 @@ public void start() {
         context.initialize();
```

Comment:
Can you please create a lifecycle object first ? There is some setup stuff in the context classes that does not belong there and if this code is moved to the lifecylce class then this approach fails badly.